eox_nelp/views.py

```diff
@@ -6,11 +6,17 @@
 import json
 from os.path import dirname, realpath
 from subprocess import CalledProcessError, check_output
-
 import six
-from django.http import HttpResponse
 
+from django.conf import settings
+from django.http import HttpResponse, JsonResponse
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import cache_page
+from rest_framework import status
+
+from lms.djangoapps.mfe_config_api.views import MFEConfigView
```

Comment:
this should be a new backend

Reply:
[2eae781](https://github.com/eduNEXT/eox-nelp/pull/17/commits/2eae78173dbf22725459db90c93f7adbe44f5168)